po/zh_CN.po

```diff
@@ -2598,7 +2598,7 @@ msgstr "磁盘"
 
 #: pkg/metrics/metrics.jsx:794
 msgid "Disks usage"
-msgstr "磁盘使用"
+msgstr "磁盘使用率"
```

Comment:
These both translate to the same "Disk usage" in Google translate. I copied this to https://translate.fedoraproject.org/translate/cockpit/main/zh_CN/ . We don't use PRs to manage translations, but that weblate instance.